package/MDAnalysis/lib/formats/libdcd.pxd

```diff
@@ -0,0 +1,147 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
```

Comment:
since this is a glorified `.h` file, would it make more sense to place it with the other one we have?

Reply:
As in with the actual format stuff in `/src` or in `libmdanalysis`?